test/unit/specs/components/governance/LiProposal.spec.js

```diff
@@ -1,80 +1,134 @@
-import setup from "../../../helpers/vuex-setup"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
+import { shallowMount, createLocalVue } from "@vue/test-utils"
 import LiProposal from "renderer/components/governance/LiProposal"
 
-const { proposals, tallies } = lcdClientMock.state
-const proposal = proposals[`2`]
-
-const $store = {
-  commit: jest.fn(),
-  dispatch: jest.fn(),
-  getters: {
-    proposals: { proposals, tallies }
+const proposals = {
```

Comment:
I would move this away and import the same from every spec file that needs it, so if the payload changes, we fix a vuex module with his test file, we have to align all the ones that uses this payload with the new version of it

Reply:
ok will do.